infrastructure/configuration/devops-agents/build.pkr.hcl

```diff
@@ -1,21 +1,67 @@
+packer {
+  required_plugins {
+    azure = {
+      source  = "github.com/hashicorp/azure"
+      version = "~> 2"
+    }
+  }
+}
+
+source "azure-arm" "azure-agents" {
+  client_id       = var.client_id
+  client_secret   = var.client_secret
+  tenant_id       = var.tenant_id
+  subscription_id = var.subscription_id
+}
+
 build {
   name = "azure-devops-agents"
 
   source "source.azure-arm.azure-agents" {
     managed_image_resource_group_name = var.resource_group_name
-    managed_image_name                = "${var.image_prefix}-${formatdate("YYYYMMDDhhmmss",timestamp())}"
+    managed_image_name                = "devops-agents-${formatdate("YYYYMMDDhhmmss",timestamp())}"
 
     os_type         = "Linux"
     image_publisher = "canonical"
     image_offer     = "0001-com-ubuntu-server-focal"
     image_sku       = "20_04-lts"
 
-    location        = var.location
+    location        = "UK South"
     vm_size         = "Standard_D2ds_v5"

```

Comment:
This VM may need to be changed as last time on ODT we ran into issues and had to change VM - brief look at documentation seems to be OK